web-marketplace/src/pages/Projects/AllProjects/AllProjects.CommunityFilter.tsx

```diff
@@ -8,9 +8,9 @@ import InfoTooltipWithIcon from 'web-components/src/components/tooltip/InfoToolt
 import { useCommunityProjectsAtom } from 'lib/atoms/projects.atoms';
```

Comment:
a good part of the changes here is about moving the existing Projects component to Projects/AllProjects so eventually we have the following structure:
- Projects
  - AllProjects tab content
  - PrefinanceProjects tab content